Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN opam config exec -- dune build ./_build/install/default/bin/opam-repo-ci-ser
 FROM debian:11
 RUN apt-get update && apt-get install libev4 openssh-client curl gnupg2 dumb-init git graphviz libsqlite3-dev ca-certificates netbase gzip bzip2 xz-utils unzip tar -y --no-install-recommends
 RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
-RUN echo 'deb [arch=amd64] https://download.docker.com/linux/debian bullseye stable' >> /etc/apt/sources.list
+RUN echo 'deb [arch=arm64] https://download.docker.com/linux/debian bullseye stable' >> /etc/apt/sources.list
```

Comment:
Could we remove this `arch` setting completely and allow a build on various platforms? It should work similar to `ocluster-worker` which builds on various platforms.
I'm working on a docker-compose.yml for local development that would also benefit from not having this.